example/settings.py

```diff
@@ -65,7 +65,7 @@
     'two_factor',
     'example',
 
-    'debug_toolbar',
+    # 'debug_toolbar',
```

Comment:
Did you try uncommenting the middleware to see if it now works under Django 1.10?